lib/internal/util/inspect.js

```diff
@@ -309,7 +310,8 @@ function inspect(value, opts) {
       ctx.showHidden = opts;
     } else if (opts) {
       const optKeys = ObjectKeys(opts);
-      for (const key of optKeys) {
+      for (let i = 0; i < optKeys.length; ++i) {
+        const key = optKeys[i];
```

Comment:
```suggestion
      for (const key of optKeys) {
```
A closer look says that this is the only part of this patch that is executed in the `util` benchmark. I know the tests are gonna fail but there's no harm in benchmarking this.

Reply:
Indeed, the only perf regression is now (and it might be a false positive):

```
 util/inspect.js option='showHidden' method='String_boxed' n=20000            ***     -7.18 %       ±3.23% ±4.29%  ±5.59%
```

That's a bummer, we really are one instruction away from performance.

Reply:
We can try a `for..in` loop, I'd be surprised if it performs well, but we might as well try it at this point.
```js
      for (const key in opts) {
        // TODO(BridgeAR): Find a solution what to do about stylize. Either make
        // this function public or add a new API with a similar or better
        // functionality.
        if (
          ObjectPrototypeHasOwnProperty(inspectDefaultOptions, key) ||
          key === 'stylize') {
          ctx[key] = opts[key];
        } else if (
          ObjectPrototypeHasOwnProperty(opts, key) &&
          ctx.userOptions === undefined) {
          // This is required to pass through the actual user input.
          ctx.userOptions = opts;
        }
      }
```

Reply:
Added it :crossed_fingers: